tests/test_copy_move.cpp

```diff
@@ -175,14 +175,17 @@ TEST_SUBMODULE(copy_move_policies, m) {
     m.attr("has_optional") = false;
 #endif
 
-    // #70 compilation issue if operator new is not public
+    // #70 compilation issue if operator new is not public - body added but not needed on most compilers
```

Comment:
This comment deserves a longer explanation. Which compilers behave oddly, is the first thing that popped into my mind when reading this code.

I know we know which ones today, but will we still remember in a year?

Reply:
MSVC and nvcc don't like having a definition with no body, even if it is never used. Having a body is better than manually trying to hide the warning, IMO, and when you start trying to hide the warning from multiple compilers, that explodes. (will add something like this to the note)

Reply:
I could probably move this out of the test definition, then the compilers would be okay with having a declaration without a definition. What they are complaining about is that these are local, so there's never a way to add the method body.

Reply:
I added the longer note; let me know if you think it's a good idea to move it up and out of the test body, so compilers will not complain about the missing definition - then we should be able to leave it empty.